content/en/docs/languages/java/sdk.md

```diff
@@ -1371,11 +1368,21 @@ The [span exporter](#spanexporter), [metric exporter](#metricexporter), and
 - `OtlpGrpc{Signal}Exporter`s export data via OTLP `grpc`.
```

Comment:
```suggestion
- `OtlpGrpc{Signal}Exporter`, which export data via OTLP `grpc`
```

(If you agree with this change, please make the corresponding change to the previous line (which I can't seem to edit via the web i/f).